qcengine/programs/executor.py

```diff
@@ -21,23 +23,40 @@ class Config:
     def compute(self, input_data: 'ResultInput', config: 'JobConfig') -> 'Result':
         pass
 
+    @staticmethod
     @abc.abstractmethod
-    def found(self) -> bool:
+    def found(raise_error: bool=False) -> bool:
         """
         Checks if the program can be found.
+
+        Parameters
+        ----------
+        raise_error : bool, optional
+            If True, raises an error if the program cannot be found.
+
+        Returns
+        -------
+        bool
+            Returns True if the program was found, False otherwise.
         """
         pass
 
 ## Utility
 
     def get_version(self) -> str:
-        """Finds program, extracts version, returns normalized version string."""
+        """Finds program, extracts version, returns normalized version string.
+
+        Returns
+        -------
+        str
+            Return a valid, safe python version string.
+        """
         pass
 
 ## Computers
 
     def build_input(self, input_model: 'ResultInput', config: 'JobConfig',
-                    template: Optional[str] = None) -> Dict[str, Any]:
+                    template: Optional[str]=None) -> Dict[str, Any]:
```

Comment:
Very minor, but is your auto-format working just from the yapf settings in setup.cfg? I turned on `yapf` autoformatting in an IDE and noticed it did exactly the opposite of the above, transforming into `asdf: List[str] = None`. I've checked that it's properly set to use yapf, updated to yapf 0.27, and observed that it's picking up on the 119 column limit.

Reply:
Ok turns out I need to update from 0.16 (!). So we now have the spaces... Not sure I am a fan of the spaces, but I guess we go with it.